### PR TITLE
Availability api

### PR DESCRIPTION
**Pull request by Haziba**
**Notes to tester:**
Not sure how I feel about the routing. I made this all on one page in the PHP sample app and wanted to keep them consistent which is why. Unsure of a better way to do it though as Ruby's `index show new create update edit` don't really fit into what we're doing here

### CODE REVIEW

File: app/views/availability/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      <tr>
+        <th>Start Time</th>
+        <th>End Time</th>
+        <th>Participants</th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <% @available_periods.each do |period| %>
+          <? for($i = 0; $i < count($availablePeriods); $i++){ ?>
```
Comment (stephenbinns) [inline app/views/availability/index.html.erb]: Is this meant to be here, it looks like its php?

**Comment by Haziba** [comment]
@stephenbinns :mag:

File: Gemfile [hunk 2]
```diff
@@ -31,7 +31,8 @@ gem 'jbuilder', '~> 2.5'
 # gem 'capistrano-rails', group: :development
 
 gem 'omniauth-cronofy'
-gem 'cronofy'
+#gem 'cronofy'
+gem 'cronofy', git: 'https://github.com/cronofy/cronofy-ruby.git', :branch => 'availability'
```
Comment (stephenbinns) [inline Gemfile]: Have we not published the gem with this in yet?

Comment (gshutler) [inline Gemfile]: Seems not. Thought I had otherwise I wouldn't have moved to beta.

Opened https://github.com/cronofy/cronofy-ruby/pull/26

**Comment by Haziba** [comment]
@stephenbinns :mag:

**Comment by stephenbinns** [comment]
👍

**Pull request merged**